From 142bb63718cbfbfe4f7ff44ca0af73ef242d0d03 Mon Sep 17 00:00:00 2001
From: Eric Yang <Eric.Yang2@amd.com>
Date: Tue, 7 May 2019 17:29:11 -0400
Subject: [PATCH 2011/2940] drm/amd/display: Move CLK_BASE_INNER macro

This macro is specificly used by RV1. Move it to the appropriate
location.

Signed-off-by: Eric Yang <Eric.Yang2@amd.com>
Reviewed-by: Aric Cyr <Aric.Cyr@amd.com>
Acked-by: Leo Li <sunpeng.li@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dcn10/rv1_clk_mgr_clk.c   | 4 ++++
 drivers/gpu/drm/amd/display/dc/inc/hw/clk_mgr_internal.h | 3 +--
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dcn10/rv1_clk_mgr_clk.c b/drivers/gpu/drm/amd/display/dc/dcn10/rv1_clk_mgr_clk.c
index 887814c65bac..e99af225845a 100644
--- a/drivers/gpu/drm/amd/display/dc/dcn10/rv1_clk_mgr_clk.c
+++ b/drivers/gpu/drm/amd/display/dc/dcn10/rv1_clk_mgr_clk.c
@@ -36,6 +36,10 @@
 
 #include "dce/dce_clk_mgr.h"
 
+#define CLK_BASE_INNER(inst) \
+	CLK_BASE__INST ## inst ## _SEG0
+
+
 #define CLK_REG(reg_name, block, inst)\
 	CLK_BASE(mm ## block ## _ ## inst ## _ ## reg_name ## _BASE_IDX) + \
 					mm ## block ## _ ## inst ## _ ## reg_name
diff --git a/drivers/gpu/drm/amd/display/dc/inc/hw/clk_mgr_internal.h b/drivers/gpu/drm/amd/display/dc/inc/hw/clk_mgr_internal.h
index 42f72c44e6f9..53f6286b2fd6 100644
--- a/drivers/gpu/drm/amd/display/dc/inc/hw/clk_mgr_internal.h
+++ b/drivers/gpu/drm/amd/display/dc/inc/hw/clk_mgr_internal.h
@@ -49,8 +49,7 @@
 	clk_mgr->ctx->logger
 
 
-#define CLK_BASE_INNER(inst) \
-	CLK_BASE__INST ## inst ## _SEG0
+
 
 #define CLK_BASE(inst) \
 	CLK_BASE_INNER(inst)
-- 
2.17.1

